easybuild/easyconfigs/m/MONAI/MONAI-1.3.0-foss-2023a.eb

```diff
@@ -0,0 +1,76 @@
+easyblock = 'PythonBundle'
+
+name = 'MONAI'
+version = '1.3.0'
+
+homepage = 'https://monai.io/'
+description = """
+MONAI is a PyTorch-based, open-source framework for deep learning in healthcare
+imaging, part of PyTorch Ecosystem.
+"""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+github_account = 'Project-MONAI'
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('PyTorch', '2.1.2'),
+    ('PyTorch-Ignite', '0.4.11'),  # Monai 1.3.0 requires PyTorch-Ignite==0.4.11
+    ('einops', '0.7.0'),
+    ('ITK', '5.3.0'),
+    ('NiBabel', '5.2.0'),
+    ('scikit-image', '0.22.0'),
+    ('tensorboard', '2.15.1'),
+    ('torchvision', '0.16.0'),
+    ('tqdm', '4.66.1'),
+    ('Pillow', '10.0.0'),
+    ('openslide-python', '1.3.1'),
+    ('BeautifulSoup', '4.12.2'),
+    ('hatchling', '1.18.0'),
+]
+
+# install MONAI with list of 'extras', which require additional dependencies
+local_pip_extras = "nibabel,skimage,pillow,tensorboard,gdown,ignite,torchvision,itk,tqdm,lmdb,psutil,openslide,fire"
```

Comment:
* `einops` is missing (dependency is there)
* `scipy` and `pandas` are missing (SciPy-bundle pulled through PyTorch, can be added as explicit dep as well)
* `pyyaml` is missing (PyYAML pulled through PyTorch, can be added as explicit dep as well)
* `jsonschema` is missing (available in Python-bundle-PyPI)
* `pydicom` is missing (pydicom pulled through Nibabel)